recipes/recognizer/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
 
 test:
   commands:
-    - "python $PREFIX/bin/recognizer.py -v"
+    - "$PREFIX/bin/recognizer.py -v"
```

Comment:
`recognizer.py` should also work, isn't it?

Reply:
Both work, but the change was made to reflect that now the script header indicates the use of python. In the previous version, it required to explicitly call python, now it can be run directly with "recognizer.py"

Reply:
Yes, now it should work remotely